macos/snd_.ksy

```diff
@@ -1,9 +1,9 @@
 meta:
-  id: snd_
+  id: mac_os_resource_snd
```

Comment:
@tisoft Please also rename the file from `macos/snd_.ksy` to `macos/mac_os_resource_snd.ksy`, the best practice is to keep the file name same as the top level `/meta/id` to avoid confusion.

Reply:
Done.